website/content/v1.8/talos-guides/install/cloud-platforms/aws.md

```diff
@@ -240,8 +240,7 @@ worker.yaml is valid for cloud mode
 #### Create the Control Plane Nodes
 
 ```bash
-CP_COUNT=1
-while [[ "$CP_COUNT" -lt 4 ]]; do
+for CP_COUNT in {1..3}; do
```

Comment:
I believe this is `bash`-specific, but the previous `(($i++))` is also bash-specific.